ui/src/transaction.ts

```diff
@@ -221,6 +221,12 @@ export function ongoing(txKind: TxKind): boolean {
   );
 }
 
+// Middleware criterion to check for an ongoing transaction of a given TxKind.
+export function ongoingB(txKind: TxKind): (tx: Tx) => boolean {
```

Comment:
I think we can remove the `ongoing` function above now

Reply:
I am sorry, dropping it now.